staff/alex-barba/gamma-template/style/style.sass

```diff
@@ -0,0 +1,564 @@
+
+/* General Rules */
+
+html, body, header, main, section, footer, button
+    margin: 0
+    box-sizing: border-box
+    width: 100%
+    font-family: "Helvetica Neue"
+    color: #333333
+
+h1, h2, h3, h4, h5, h6, p, ul  
+    margin: 0
+
+/* Mixins */
+
+@mixin flex-space-bt
+    display: flex
+    justify-content: space-between
+
+@mixin flex-col
+    display: flex
+    flex-direction: column
+
+/* Header in Home/About/Contact*/
+
+.menu
+    +flex-space-bt
+    align-items: center
+    padding-left:1rem
+    padding-right:1rem
+    padding-top: 0.5rem
+    padding-bottom: 0.5rem
+    
+
+    &__title
+        font-size: 0.8rem
+        color: #1baa98
+
+    &__button
+        display: flex
+        width: 2rem
+        height: 1rem
+        border: 0.01rem solid #999999
+        padding: 0.5rem
+
+.nav
+    display: none
+
+/* Head in About/Contact */
+
+.head
+    background-color: #1baa98
+    font-size: 1.2rem
+    color: white
+    padding: 1rem
+    margin-bottom: 2rem
+
+/* Main */
+
+main 
+    +flex-col
+
+/* First ad in Home*/
+
+.ad
+    +flex-col
+    background-color: #1B222A
+    padding-bottom: 2rem
+    padding-top: 1.5rem
+    
+    &__img
+        display: none
+
+.title
+    +flex-col
+    justify-content: center
+
+    &--white
+        color: #ffffff
+        text-align: center
+        padding-top: 1rem
+        font-size: 2rem
+    
+    &--green
+        color: #1baa98
+
+    &__body
+        color: white
+        text-align: center
+        padding: 1.5rem
+        font-size: 1.2rem
+
+    &__img
+        +flex-col
+
+    &__img img
+        width: 55%
+        align-self: center
+        padding-bottom: 1rem
+
+/* About in About */
+.content
+    padding-left:1rem
+    padding-right:1rem 
+
+    &__about h3
+        padding-bottom: 0.5rem 
+    
+    &__about p
+        padding-bottom: 0.5rem
+
+    &__card
+
+.collapsible
+    font-size: 0.9rem
+    border: 0.01rem solid #999999
+    background-color: #F0F0F2
+    padding: 0.3rem
+    padding-left: 0.5rem
+    text-align: start 
+    font-weight: bold 
+    margin-top: 0.3rem
+
+    &__content
+        font-size: 0.8rem
+        padding-left: 0.5rem
+        border: 0.01rem solid #999999
+        border-top: 0
+
+/* Form in Contact */
+
+.content
+    &__form
+        padding-right: 1rem
+
+    &__form p 
+        font-weight: bold
+        font-size: 0.8rem
+    
+    &__form input
+        font-size: 0.8rem
+        padding-top: 0.3rem
+        padding-bottom: 0.3rem
+        padding-left: 0.5rem
+        border: 0.01rem solid #999999
+        width: 100%
+        color: #999999
+        margin-bottom: 0.5rem
+
+    &__form textarea
+        font-size: 0.8rem
+        padding-top: 0.3rem
+        padding-bottom: 0.3rem
+        padding-left: 0.5rem
+        border: 0.01rem solid #999999
+        width: 100%
+        color: #999999
+        margin-bottom: 0.5rem
+    
+.btn__submit 
+    border: 0.01rem solid #999999 !important
```

Comment:
You should try not to use !important, cause if in the future you need to change something of that class it would be very difficult.

Reply:
I know, but it was the only way I found out...